src/sagemaker/image_uri_config/huggingface.json

```diff
@@ -913,7 +913,7 @@
                         "us-west-2": "763104351884"
                     },
                     "repository": "huggingface-pytorch-inference",
-                    "container_version": {"gpu": "cu111-ubuntu20.04", "cpu": "ubuntu20.04" }
+                    "container_version": {"gpu": "cu111-ubuntu20.04", "cpu": "ubuntu20.04", "neuron": "sdk1.17.1-ubuntu18.04-v1.0" }
```

Comment:
How would this work if release a new DLC for neuron SDK `1.18.0` for the same framework version?

Reply:
addressed in current revision, by including new key/value pairs in the huggingface-neuron.json file

Reply:
If sdk version is not passed as input argument, then it the retrieve will pick the latest sdk version